### PR TITLE
when user register then create report

### PR DESCRIPTION
**Pull request by nguyenhonghai1997bg**

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -68,12 +79,33 @@ public function create()
      */
     public function store(UserFormRequest $request)
     {
-        $data = $request->all();
-        $data['password'] = bcrypt($request->password);
-        $data['status'] = 0;
-        $data['role'] = $this->roleRepository->getIdTrainee()->id;
-        $user = $this->userRepository->create($data);
-        Alert::success(trans('Register Member Successfully'), trans('Please Wait Active'));
+        DB::Transaction(function () use ($request){
```
Comment (framgiaciv3) [inline app/Http/Controllers/UserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.SpaceBeforeBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 82</br>Expected 1 space before opening brace; found 0

File: app/Repositories/BatchRepository.php [hunk 2]
```diff
@@ -10,4 +10,16 @@ public function model()
     {
         return \App\Models\Batch::class;
     }
+
+    public function listBatchesArray()
+    {
+        $array = [];
+        $results = $this->model->all();
+        foreach ($results as $result) {
+            $array[$result->id] =
```
Comment (framgiaciv3) [inline app/Repositories/BatchRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/BatchRepository.php</br>**Line**: 19</br>Whitespace found at end of line

File: app/Repositories/BatchRepository.php [hunk 3]
```diff
@@ -10,4 +10,16 @@ public function model()
...
         return \App\Models\Batch::class;
     }
+
+    public function listBatchesArray()
+    {
+        $array = [];
+        $results = $this->model->all();
+        foreach ($results as $result) {
+            $array[$result->id] = 
+                $result->batch . '-' . $result->workspace->name . '-' . $result->program->name . '-' . $result->position->name;
```
Comment (framgiaciv3) [inline app/Repositories/BatchRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.Files.LineLength.TooLong</br>**Severity**: warning</br>**File**: app/Repositories/BatchRepository.php</br>**Line**: 20</br>Line exceeds 120 characters; contains 127 characters

**Comment by nguyenhonghai1997bg** [comment]
https://edu-redmine.framgia.vn/issues/5421

**Pull request merged**